Update Employee_Database_Script.sql

harjot85 opened the pull request:
Made following changes in the style for enhanced readability. (I am a student, and it is our assignment to help improve code/documentations on GitHub)
- Indented the code so it looks unified. 
- Capitalized Keywords and constraints, so they stand out. 
- Separated the field names with an underscore ( _ ), to make the name readable easily.

harjot85 committed:
Update Employee_Database_Script.sql

Made following changes in the style for enhanced readability. (I am a student, and it is our assignment to help improve code/documentations on GitHub)
- Indented the code so it looks unified. 
- Capitalized Keywords and constraints, so they stand out. 
- Separated the field names with an underscore ( _ ), to make the name readable easily.

tolgahanakgun commented:
@harjot85  Thanks for feedback and helping to improve the code. I want the attribute names and table names to be like names in the Fundamentals of Database Systems 6th Edition, so all example sql queries in the book can be executed without any change. If you can modify the code according to the mentioned book style and structure then i will merge.

harjot85 commented:
No problem, I'll get on it.

harjot85 committed:
Update Employee_Database_Script.sql

I have modified the code according to the style and structure of the attribute names and table names that matches the style in the Fundamentals of Database Systems 6th Edition.
